### ISSUE TITLE
Unable to find `eslint-plugin-local` without `.git` folder

### ISSUE THREAD

**Issue by ssche** [taskworld/eslint-plugin-local#6]
The code below assumes that the root directory is where a `.git` directory is located. This assumes:

* that the user is using git
* that the application that is using this plugin is located at the root directory. A structure such as
  ```
   foo
     +--- .git
     +--- app
           +--- webapp
                   +--- node_modules
             .eslintrc.js
             .eslint-plugin-local.js
             package.json
             ...
  ```
  would not work.

Can we find a better solution to locate `eslint-plugin-local.js`? For instance, it could always be assumed to be where `node_modules` is located (in the same folder that contains `node_modules`).

```
function findRootPath(workPath) {
	if (workPath === path.dirname(workPath)) {
		return null
	}

	const testPath = path.join(workPath, '.git')
	if (fs.existsSync(testPath) && fs.lstatSync(testPath).isDirectory()) {
		return workPath
	}

	return findRootPath(path.dirname(workPath))
}
```

**Comment by ssche** [comment]
Actually, I subsequently found [this PR](https://github.com/taskworld/eslint-plugin-local/pull/5) which would resolve the issue. Please merge into master and release if possible.

**Comment by ThisIsManta** [comment]
Hi there! This is supposed to be fixed in [v3.2.0](https://github.com/taskworld/eslint-plugin-local/releases/tag/v3.2.0).
Please refer to https://github.com/taskworld/eslint-plugin-local/pull/5#issuecomment-1925435708

Cheers!
